extensions/amp-analytics/0.1/vendors.js

```diff
@@ -442,3 +442,5 @@ export const ANALYTICS_CONFIG = {
 ANALYTICS_CONFIG['infonline']['triggers']['pageview']['iframe' +
 /* TEMPORARY EXCEPTION */ 'Ping'] = true;
 
+ANALYTICS_CONFIG['adobeanalytics']['triggers']['pageview']['iframe' +
```

Comment:
This breaks existing users. Lets rename the request, so users can migrate to the iframe pageview ping if they want to.